### ISSUE TITLE
Substract percent incorrect

### ISSUE THREAD

**Issue by oleksiivasylenko** [serso/android-calculatorpp#222]
Hi, version of app 2.2.3

Using app:
0.0003 - 0.1% = 0.0003
0.0003 - 1% = 0.0003

Correct:
0.0003 - 0.1% = 0.0002996..
0.0003 - 1% = 0.0002969..

of course there is precision, but still 0.0003 - 1% = 0.0003 this looks totaly wrong
![photo_2024-03-04_15-05-32](https://github.com/serso/android-calculatorpp/assets/28143505/7db85878-c864-4cac-ac7a-8b8bb94a2b1b)

**Comment by derei** [comment]
@oleksiivasylenko you are correct: the issue is your precission settings. The result is fine, considering what precission settings you may have.

Same operation on my end:
![Percent_substraction](https://github.com/user-attachments/assets/178c81f1-3f83-458f-92ac-a3b7a3a91558)

**Comment by oleksiivasylenko** [comment]
> Same operation on my end:

Thanks! Could you please let me know where I can set this setting? And the issue is reloved

**Comment by derei** [comment]
@oleksiivasylenko 
![ImgTbox_2024-07-20_15-08-01_1606](https://github.com/user-attachments/assets/9d1aaae1-d6ed-47e3-91a0-16baca2fd29d)
